controller/volume_controller.go

```diff
@@ -127,6 +130,11 @@ func NewVolumeController(
 
 	vc.scheduler = scheduler.NewReplicaScheduler(ds)
 
+	nodeInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc:    vc.enqueueVolume,
```

Comment:
This is nodeInformer callback, why you use existing enqueueVolume?

Reply:
Originally I thought this would trigger enqueueVolume but seems like it's a no-op so I've removed the nodeInformer. However, I don't know what is constantly triggering this yet.

Reply:
The informers get resynced at per 30s interval [here](https://github.com/longhorn/longhorn-manager/blob/master/controller/controller_manager.go#L64-L71). After discussing with @innobead we might just leverage this behavior and will not add in nodeInformer since it might not be worth to add in those extra volume queues.